tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -1084,6 +1084,27 @@ private void writeSummaryTable(
 
     private Map<String, String> getSortedNames(Organization org, Level desiredLevel) {
         Map<String, String> sortedNames = new TreeMap<>(CLDRConfig.getInstance().getCollator());
+        LocalesWithExplicitLevel cldrLocale =
+                new LocalesWithExplicitLevel(Organization.cldr, desiredLevel);
+
+        // TODO: another hack, unaffiliated gets everything
```

Comment:
"TODO" should have a ticket number and clarify what needs to be done, otherwise just a comment not a "TODO"

Reply:
The todo is not needed here. I'll remove it.

Reply:
removed